keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/model/ActiveServerGroup.kt

```diff
@@ -173,6 +212,23 @@ data class LaunchConfig(
   val instanceMonitoring: InstanceMonitoring
 )
 
+data class LaunchTemplate(
+  val launchTemplateData: LaunchTemplateData
+)
+
+data class LaunchTemplateData(
```

Comment:
I am not sure what other `LaunchTemplateData` keel cares about, but here is a [link ](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-launchtemplate-launchtemplatedata.html)to the other properties just in case.

Reply:
I wasn't sure whether to populate all of the fields or just the ones we are crrently consuming, so I started with just the ones we are currently consuming.

Also, I see it supports ram disk id! I didn't see that in the payloads that I was playing with, I'm going to add that in for parity.

Reply:
This models the `aws` `launchTemplateData`,  are the other properties dropped? 
I guess you only need properties that you are specifically accessing on the object.

Reply:
@jeyrschabu Yeah, this initial implementation just parses out the data that keel consumes, and ignores the other fields.